components/AddLostItem.js

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+import styled from "styled-components";
+import { useState } from "react";
+import { lostItems } from "./Db";
+import { nanoid } from "nanoid";
+import Image from "next/image";
+
+export default function AddLostItem() {
+  const [description, setDescription] = useState("");
+  const [location, setLocation] = useState("");
+  const [name, setName] = useState("");
+  const [lost, setLost] = useState(lostItems);
+  const [img, setImage] = useState("/assets/MrBean.png");
+
+  const randomId = nanoid();
+
+  const handleClick = (event) => {
+    event.preventDefault();
+    const newLostItem = { name, description, location, img, randomId };
```

Comment:
const formdata = new FormData(event.target);
const data = Object.fromEntries(formdata);
const newLostItem = {...data,id: Math.random().toString(36).substring(2)}

Reply:
@anhthu253 where will the state be used in this example? I think the " const [lost, setLost] = useState(lostItems);" hast to remain in my code to use this data on another page (after lifting the state).